request.py

```diff
@@ -0,0 +1,127 @@
+import logging
+from datetime import datetime
+from sqlalchemy import exc
+import sqlalchemy
+from flask import jsonify, request, make_response, g
+from init import app
+import function
+# import utils
+from models import User
+from flask_httpauth import HTTPBasicAuth
+auth = HTTPBasicAuth()
+
+
+@app.route("/")
+def index():
+    return "Welcome to Hangman!"
```

Comment:
If you want you can. This endpoint is not very useful though. Eventually if you want you can use something like swagger to document your API and give a map of all your functions available. But mapping index is not mandatory and doesn't really bring much value to your RESTful API.